client/index.html

```diff
@@ -117,7 +117,7 @@ <h2 class="text-center">Input your data</h2>
       <h2 class="text-center">View your data</h2>
       <p class="text-center">Your data will appear here after you drag/drop or browse to find your completed Excel
         template file above.</p>
-      <span id="expand-table-button" class="arrow-down flip"></span>
+      <span id="expand-table-button" class="arrow-down"></span>
```

Comment:
This will break for BWWC, since their tables are supposed to start collapsed. I guess this should become a config option?